src/utils/date.ts

```diff
@@ -0,0 +1,16 @@
+import { DateTime } from "luxon";
+
+export interface DateAsStrings {
+    day: string,
+    month: string,
+    year: string
+}
+
+export const getTodaysDate = (): DateAsStrings => {
+  const now = DateTime.now().toUTC();
+  return {
+    day: "" + now.day,
```

Comment:
With regard to that other structure (InputDate), I've seen other functions that pad the values assigned - e.g. "01" instead of "1" for the first day of the month. Is that relevant here?

Reply:
the template uses a macro which pads and converts month to eg "January' so we just needed to get the numeric values for todays date into the template

Reply:
Ok. Nice.